fpdf/html.py

```diff
@@ -223,6 +224,12 @@ def __init__(
         self.table_line_separators = table_line_separators
         self.ul_bullet_char = ul_bullet_char
         self.style = dict(b=False, i=False, u=False)
+        self.leading_space = re.compile(r"^\s+")
```

Comment:
As those 3 RegExps are constants, we should define them in UPPERCASE, at a module or class level